web/html/src/components/states-picker.tsx

```diff
@@ -0,0 +1,447 @@
+import * as React from "react";
+
+import _partition from "lodash/partition";
+import _sortBy from "lodash/sortBy";
+import _unionBy from "lodash/unionBy";
+
+import { SectionToolbar } from "components/section-toolbar/section-toolbar";
+
+import { DEPRECATED_unsafeEquals } from "utils/legacy";
```

Comment:
Please avoid using this for new code. To get rid of it, you can check if this call can be replaced with `===` with correct types, e.g. `DEPRECATED_unsafeEquals(a, b)` into `a === b` and if not, figure out a way to make it comparable so we don't compare `false` to `[]` etc.